client/src/modules/StudentDashboard/components/TasksChart.tsx

```diff
@@ -1,45 +1,81 @@
 import { Pie, PieConfig } from '@ant-design/plots';
+import { getTaskStatusColor } from 'modules/Schedule/components/TableView/renderers';
```

Comment:
If you are reusing renderer from Schedule, it's better to move it to components\Table\renderers.tsx (common folder for tables) and import it from there

Reply:
disagree. this logic is not common component, it strictly relates to tasks and as of now to schedule. the fact it is used in the dashboard is fine since it still relates to schedule tasks. 
reworked in a different way to export properly

Reply:
Okay then, thank you